terraform/modules/redis/main.tf

```
@@ -20,6 +20,14 @@ resource "aws_security_group" "redis" {
   name        = "${var.cluster_name}_elasticache"
   vpc_id      = var.vpc_id
   description = "Access to the ${var.cluster_name} Redis cluster"
+
+  tags = merge(
+    var.additional_tags,
+    {
+      name = "$var.cluster_name}-redis-sg"
```

Missing an open brace here
```suggestion
      name = "${var.cluster_name}-redis-sg"
```

---

good spot - i will correct this